main.py

```diff
@@ -2810,12 +2821,12 @@ async def lb_handler(interaction, type, do_edit=None):
                 messager = round(messager / 3600, 2)
 
         # dont show placements if they arent defined
-        if interactor and (type == "Cats" or type == "Slowest") and interactor <= 0:
+        if interactor and (type == "Cats" or type == "Slowest"or type == "Value") and interactor <= 0:
```

Comment:
buddy ate the space

Reply:
okay fine ill make it consistent

Reply:
if(interactor)and(type=="Cats"or(type)=="Slowest"or(type)=="Value")